nylas/models/events.py

```diff
@@ -309,6 +310,7 @@ class Event:
     id: str
     grant_id: str
     calendar_id: str
+    capacity: int
```

Comment:
Capacity should be optional, can you mark it as so and move it down with the optional fields? Thanks!